fabric-common/lib/ServiceAction.js

```diff
@@ -59,14 +59,14 @@ const ServiceAction = class {
 	 *  When the 'param' is a byte[], the bytes will be used as the final
 	 *  commit signature.
 	 */
-	sign(param = checkParameter('param')) {
+	async sign(param = checkParameter('param')) {
```

Comment:
JSDoc and type definitions also need to be updated to reflect this change

Reply:
The type definitions in `fabric-common/types/index.d.ts` also need to be changed to match the code.